freqtrade/exchange/bybit.py

```diff
@@ -233,3 +234,25 @@ def fetch_order(self, order_id: str, pair: str, params: Dict = {}) -> Dict:
             # Canceled orders will have "remaining=0" on bybit.
             order['remaining'] = None
         return order
+
+    def cancel_order_with_result(self, order_id: str, pair: str, amount: float) -> Dict:
+        order = super().cancel_order_with_result(order_id, pair, amount)
+
+        # Currently bybit does not update status of the cancelled order on time.
+        # Even though endpoint returns proper response and code 200, the status stays open.
+        # This re-fetches the order giving bybit enough time to update the status.
```

Comment:
I'm not entirely sure this is true.

i've seen delays in order status updates of up to 8 minutes (very high volume day) on bybit (though on the old API version) - so this will not actually give you any guarantee that the status effectively changes.